src/status_im/contexts/wallet/temp.cljs

```diff
@@ -105,3 +105,23 @@
                 :type         :network
                 :network-logo (quo.resources/get-network :ethereum)
                 :network-name "Mainnet"}}])
+
+(def collectible-about
+  {:title "Bored Ape Yacht Club"
```

Comment:
I think we could leave only links info in temp, since other data is available in `status-mobile` already

Reply:
Done